src/hotspot/share/classfile/vmSymbols.hpp

```diff
@@ -487,6 +487,7 @@
   template(use_unaligned_access_name,                 "UNALIGNED_ACCESS")                         \
   template(data_cache_line_flush_size_name,           "DATA_CACHE_LINE_FLUSH_SIZE")               \
   template(during_unsafe_access_name,                 "during_unsafe_access")                     \
+  template(no_deduplication_name,                     "no_deduplication")                         \
```

Comment:
This addition was left over from an earlier version of the change, where I'd added an ordinary private boolean member of this name to String.  That was before I learned about injected members and before the idea of adding "duplication requested" came up.  This name is no longer needed, and I'll be removing it.